src/datasources/cache/cache.first.data.source.spec.ts

```diff
@@ -58,6 +58,7 @@ describe('CacheFirstDataSource', () => {
       cacheDir,
       url: targetUrl,
       notFoundExpireTimeSeconds,
+      expireTimeSeconds: faker.number.int(),
```

Comment:
As before regarding a `min` option.

Reply:
Fixed in [1676ade](https://github.com/safe-global/safe-client-gateway/pull/1215/commits/1676ade45f02da84872f938601a84b03163a93b0)